docs/develop.md

```diff
@@ -89,12 +93,14 @@ spec:
     http:
       paths:
       - backend:
-          serviceName: httpserver
+          serviceName: httpbin.default.svc.cluster.local
```

Comment:
I found the port of httpbin container is 80, the port should be change to `80` instead of `8080`.
Or we do need a `service` file to describe the port Mapping.

Reply:
OK, let me add the service file.

Reply:
Fixed @gxthrj .